### ISSUE TITLE
Arctic Foxes (New Card)

### ISSUE THREAD

**Issue by kelsioboy** [magarena/magarena#1301]
name=Arctic Foxes
image=http://magiccards.info/scans/en/ia/226.jpg
value=1.854
rarity=C
type=Creature
subtype=Fox
cost={1}{W}
pt=1/1
ability=As long as defending player controls a snow land, SN can't be blocked by creatures with power 2 or greater.
timing=main
oracle=Arctic Foxes can't be blocked by creatures with power 2 or greater as long as defending player controls a snow land.
